integration/update_field_compat_test.go

```diff
@@ -709,6 +794,9 @@ func TestUpdateFieldCompatSet(t *testing.T) {
 func TestUpdateFieldCompatSetArray(t *testing.T) {
 	t.Parallel()
 
+	stringsProvider := shareddata.AllProviders().
+		Remove("ArrayDoubles", "ArrayInt32s", "ArrayRegexes")
```

Comment:
It looks to me that if we want to specify string providers, they should be included, not excluded. Otherwise, every time we add a new (non-string) provider we need to come to this part of the code and exclude it here.

Also, do we need to specify them? Can't we ignore Tigris-related tests/errors?

Reply:
Yes that's an option, let me ignore errors :+1: